.github/ISSUE_TEMPLATE/design_or_architecture.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Software Design
+about: 
+title: 'Design ...'
+labels: 'T-design'
+assignees: ''
+---
+
+## Scope 
+<!-- Clarify what the design or architecture should cover (e.g., a system, feature, component, or entire product). -->
+
+## Functional Requirements
+<!-- A list of functional specifications the system or feature must fulfill. Use cases or user stories demonstrating expected behavior. -->
+
+## Non-Functional Requirements
+<!-- If applicable provide Performance, Scalability, Security or other metrics and characteristics of module to design -->
+
+
+## Related resources
+<!-- List resources that might be useful during design/research -->
+
+
+## Acceptance criteria
+<!-- Acceptance criteria in terms of design or architecture usually (but not only) should consist of:
+- list of open questions that need to be answered 
+- some summary, research paper or meta knowledge as an outcome of research or investigation
+- at least two people sign off. One of them is the Tech lead second should be decided with team
+- workshop or knowledge sharing session should be conducted to share outcomes
+- Software design artifacts provided (eg  flow chart, use case, pseudocode)
+-->
+[] add question here..
+[] Summary, research paper or meta knowledge written and merged to main repo
```

Comment:
How is this different from the "artifacts produced"?

IMHO we should keep the scope of this template narrow and concrete, oriented on the parachain design work. That is, usually the artifacts are a design document reviewed and merged to the repo, plus issues that break up the document into estimatable chunks that should result in reviewable PRs.

For everything that doesn't fit this mold, we can use investigation issues.

Reply:
fixed